Example/AppDelegate+CarPlay.swift

```diff
@@ -553,6 +553,9 @@ class CarPlaySceneDelegate: NSObject, CPTemplateApplicationSceneDelegate {
         appDelegate.carPlayManager.templateApplicationScene(templateApplicationScene,
                                                             didConnectCarInterfaceController: interfaceController,
                                                             to: window)
+        if appDelegate.currentAppRootViewController?.activeNavigationViewController != nil {
```

Comment:
This change fixes problem for CarPlay devices that are based on iOS 13 and higher. Can you please make sure that this issue doesn't occur on iOS versions lower than iOS 13? `application(_:didConnectCarInterfaceController:to:)` will be called in this case.